plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesInPluginXmlTest.java

```diff
@@ -40,14 +42,22 @@
   private static final String OBJECTIFY_LIBRARY_ID = "objectify";
   private static final String SERVLET_API_LIBRARY_ID = "servlet-api-2.5";
   private static final String JSP_API_LIBRARY_ID = "jsp-api-2.1";
+  private static final String GUAVA_LIBRARY_ID = "guava-20";
 
   @Test
   public void testLibrarySize() {
-    assertThat(CloudLibraries.getLibraries("appengine").size(), is(3));
+    assertThat(CloudLibraries.getLibraries("appengine").size(), is(4));
+
+    // There may be 2 or 4 servlet libraries depending on whether the
+    // .appengine.java.standard.java8 bundle is present
     List<Library> servletLibraries = CloudLibraries.getLibraries("servlet");
-    // todo in Eclipse this returns 4 (versions 2 and 3 of JSP and servlet libs)
-    // figure out why
-    assertEquals("wrong number of servlet libraries", 2, servletLibraries.size());
+    // split out toString() matches because of type inference errors
```

Comment:
This change seems worth including, irrespective of the rest.

Reply:
Will do.